apps/src/lab2/views/dialogs/StartOverDialog.tsx

```diff
@@ -18,16 +18,21 @@ const StartOverDialog: React.FunctionComponent<BaseDialogProps> = ({
   );
   const isTextWorkspace =
     currentAppName && TEXT_BASED_LABS.includes(currentAppName);
-
+  let dialogMessage = commonI18n.startOverWorkspace();
+  if (isTextWorkspace) {
```

Comment:
This feels a bit like it could get hairy fast...could we do something like an object that tracks lab-specific messages, then have a fallback to text vs. non-text? Something like 

```
const LAB_SPECIFIC_MESSAGES = {
  aichat: commonI18n.startOverAichatModelCustomizations()
};

const dialogMessage = LAB_SPECIFIC_MESSAGES[currentAppName] || 
  isTextWorkspace ?
    commonI18n.startOverWorkspaceText() :
    commonI18n.startOverWorkspace();
```

Reply:
Oh nice! Will update.

Reply:
Had to circumvent a TS error so updated a slightly different way. Lmk what you think @bencodeorg.
TIL About `Partial` typing! [doc](https://www.typescriptlang.org/docs/handbook/utility-types.html)